prow/plugins/verify-owners/verify-owners.go

```diff
@@ -483,7 +483,8 @@ func nonTrustedUsersInOwners(ghc githubClient, log *logrus.Entry, triggerConfig
 // and then checks if the owner is a trusted user.
 func checkIfTrustedUser(ghc githubClient, log *logrus.Entry, triggerConfig plugins.Trigger, owner, patch, fileName, org, repo string, nonTrustedUsers map[string][]string, repoAliases repoowners.RepoAliases) (map[string][]string, error) {
```

Comment:
Do we have unit tests that would capture this change in behavior?

Reply:
We do now ;-)